spaceapi_server/src/lib.rs

```diff
@@ -87,6 +89,21 @@ impl SpaceapiServer {
         }
     }
 
+    fn update_values(&self, map: &urlencoded::QueryMap) -> IronResult<Response> {
+        // store data to datastore
+        let datastore_clone = self.datastore.clone();
+        let datastore_lock = datastore_clone.lock().unwrap();
+        println!("{:?}", map);
+
+        for item in map.iter() {
+            // TODO: check if key exists and handle errors
```

Comment:
We should also check whether the key is a valid sensor data key, once the other pull request is merged :)